x-pack/test/security_solution_cypress/cypress/e2e/detection_response/detection_engine/rule_creation/eql_rule_suppression_sequence.cy.ts

```diff
@@ -36,11 +41,8 @@ describe(
         fillDefineEqlRule(rule);
       });
 
-      it('disables the suppression fields and presents an informative tooltip', () => {
-        cy.get(ALERT_SUPPRESSION_FIELDS_INPUT).should('be.disabled');
-
-        cy.get(ALERT_SUPPRESSION_FIELDS).trigger('mouseover');
-        cy.get(TOOLTIP).contains('Suppression is not supported for EQL sequence queries.');
+      it('displays the suppression fields', () => {
```

Comment:
At the time I was focused on ensuring the fields were present and wanted to spend time developing the FTR tests. I can enhance this test to at the very least proceed to the next step of the rule creation process. Would that be sufficient or is there a preference to go further and check that the rule ran successfully after creation? I just figured one less complex cypress test the better, since most of the alert generation functionality is covered by the FTR tests.